### ISSUE TITLE
Remove dependency on Python 2.7.x

### ISSUE THREAD

**Issue by chmanie** [JoinColony/colonyNetwork#1034]
## Summary
tl;dr: 
- Python 2.x is EOL
- `truffle-security` depends on the `sleep` module which is old and required Python 2.x to build
- `truffle-security` itself is deprecated and no longer maintained

## Steps to Reproduce (for bugs)
Clone the repo and run `yarn`

## Expected Behavior
Packages are installed properly when Python 2.x is not available

## Current Behaviour
An error is thrown when building `sleep`:
```
colonyNetwork/node_modules/truffle-security/node_modules/sleep: Command failed.
Exit code: 1
Command: node-gyp rebuild
Arguments: 
Directory: /home/chris/dev/colony/colonyNetwork/node_modules/truffle-security/node_modules/sleep
Output:
gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@12.16.3 | linux | x64
gyp ERR! configure error 
gyp ERR! stack Error: Command failed: /usr/bin/python -c import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack   File "<string>", line 1
gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gyp ERR! stack SyntaxError: Missing parentheses in call to 'print'. Did you mean print(...)?
gyp ERR! stack 
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:303:12)
gyp ERR! stack     at ChildProcess.emit (events.js:310:20)
gyp ERR! stack     at maybeClose (internal/child_process.js:1021:16)
gyp ERR! stack     at Socket.<anonymous> (internal/child_process.js:443:11)
gyp ERR! stack     at Socket.emit (events.js:310:20)
gyp ERR! stack     at Pipe.<anonymous> (net.js:672:12)
gyp ERR! System Linux 5.15.35-1-lts
gyp ERR! command "node/v12.16.3/bin/node" "colonyNetwork/node_modules/.bin/node-gyp" "rebuild"
gyp ERR! cwd colonyNetwork/node_modules/truffle-security/node_modules/sleep
```

## Possible Solution
Remove `truffle-security` and use [mythx-cli](https://github.com/dmuhs/mythx-cli) instead (as proposed by the developers).

## Context
Fewer and fewer people have it Python 2.x installed as it is EOL. Fixing this would remove an additional barrier to people trying to run the code. Using `pyenv` works but is an additional burden to set up.
Furthermore it might clash with packages that use exclusive Python 3 functions in the future (I'm kind of surprised that this hasn't happened yet).
I don't know if `truffle-security` is still actively used here; in any case I would suggest to remove it and - if necessary - replace it with the better maintained `mythx`
